.claude/skills/restore-dbs.md

```
@@ -0,0 +1,125 @@
+## Restoration Commands
+
+### Restore Full Set (Recommended)
+
+Restores alkemio + synapse (kratos optional):
+
+```bash
+cd .scripts/backups && bash restore_latest_backup_set.sh <environment> [restart_services] [non_interactive] [restore_kratos]
+```
+
+**Arguments:**
+| Arg | Values | Default | Description |
```

markdownlint-cli2 (0.20.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)